CHANGELOG.md

```diff
@@ -70,6 +71,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (x/distribution) [#17657](https://github.com/cosmos/cosmos-sdk/pull/17657) The `FundCommunityPool` and `DistributeFromFeePool` keeper methods are now removed from x/distribution.
+* (x/distribution) [#17657](https://github.com/cosmos/cosmos-sdk/pull/17657) The distribution keeper now takes 2 more arguments, msg service router and grpc query router.
```

Comment:
This should be updated to taking the protocpool keeper.

Reply:
yes, I plan to update the changelog after fixing all the tests and updates of the PR